cloud_utils/slack_utils.py

```diff
@@ -35,3 +46,85 @@ async def report_exception(webhook_url: str, text: str):
         _make_exception_payload,
         gamla.post_json_async(5, webhook_url),
     )
+
+
+async def post_to_notifications_and_dm_user(
+    message: str, email: str = "", email_map: Dict[str, str] = frozendict()
```

Comment:
1. Why is email optional here? if there is no email you can call `post_to_notifications` directly.
2. Why do we need a return value? Do we have a contingency if posting fails?

Reply:
@erez-work 
1. you're right
2. the return value is useful  to inform the client whether the email is legit. AFAIK this is the only way to tell.